### DISCUSSION TITLE
When can we expect HTTP metrics export in the otel demo app?

### DISCUSSION THREAD

**Discussion by meSATYA** [open-telemetry/opentelemetry-demo#713] [General]
Otel has added http metric_exporter in its v1.15.0 release. 
I am looking to test metrics exporting in http format for the demo app. Any ETA for this integration?

**Comment by austinlparker** [comment]
Which SDK added this?

**Reply by meSATYA** [reply]
https://github.com/open-telemetry/opentelemetry-python/blob/v1.15.0/exporter/opentelemetry-exporter-otlp-proto-http/src/opentelemetry/exporter/otlp/proto/http/metric_exporter/__init__.py#L76

**Reply by Chaitan1991** [reply]
Hi @austinlparker, I was also testing the same, could you please share your thought on this or add someone to this discussion who takes care of Otel-demo app code?


Best regards,
Chetan

**Reply by puckpuck** [reply]
We try to keep all services updated with the latest SDKs, but this is a never-ending effort. Typically the changes are trivial and need just a dependency version update, but sometimes demo app code changes are also necessary due to changes in the SDK.

If this is important for you, we will certainly welcome any PRs you make to update the SDKs.

**Reply by julianocosta89** [reply]
Hello all, maybe a bit late for the discussion, but I don't see the need for this to be added to the demo.

We have as goal to make all services export Traces, Metrics and Logs. 
We have never defined the protocol which we should do that, and for most of the cases we just use the default exporter, which is gRPC.

If the service is already sending metrics, there is no need to add the HTTP Metrics exporter to the service.
